custom/icds/views/custom_sms_report.py

```diff
@@ -78,3 +92,8 @@ def _prepare_display_message(reports_in_progress, report_count):
     )
     return message
 
+
+def _report_already_in_progress(reports, start_date, end_date):
+    similar_reports=[report for report in reports
+        if report['start_date'] == start_date and report['end_date'] == end_date
```

Comment:
nit: can use `any` here. Might not be really useful here since its a small list but even then conveys the intent more clearly and also helps skip length check